apps/codestyle/codestyle-sql.py

```diff
@@ -102,10 +102,15 @@ def trailing_whitespace_check(file: io, file_path: str) -> None:
 def sql_check(file: io, file_path: str) -> None:
     global error_handler, results
     file.seek(0)  # Reset file pointer to the beginning
+    not_delete = ["creature_template", "gameobject_template", "item_template"]
     check_failed = False
 
     # Parse all the file
     for line_number, line in enumerate(file, start = 1):
+        if [match for match in ["DELETE FROM `{not_delete}`"] if match in line]:
+            print(
+                f"Entries from this tables should not be deleted! {file_path} at line {line_number}\nThis error can safley be ignored if you know what you are doing.")
```

Comment:
```suggestion
                f"Entries from this tables should not be deleted! {file_path} at line {line_number}")
```
imo, cant think of any reason to delete from those tables

Reply:
I'm sure there are a few entries in there that are custom, e.g. the ones at 3,000,000+ or whatever. Maybe those should be removed at some point, doubt they're Blizzlike.

Reply:
You can still merge even if it fails so it is fine